### PR TITLE
update pipeline guide and FAQ about OOM

### PR DESCRIPTION
**Pull request by lvhan028**
Users complains about OOM a lot

### CODE REVIEW

File: docs/en/inference/pipeline.md [hunk 1]
```diff
@@ -6,29 +6,47 @@ Then, we will describe the pipeline API in detail.
...
+# decrease the ratio of the k/v cache occupation to 20%
+backend_config = TurbomindEngineConfig(cache_max_entry_count=0.2)
+
+pipe = pipeline('internlm/internlm2-chat-7b',
+                backend_config=backend_config)
+response = pipe(['Hi, pls intro yourself', 'Shanghai is'])
+print(response)
+```
+
+A better approach would be to allocate space for the k/v cache from the free GPU memory proportionally. We will improve it in future versions for a better user experience.
```
Comment (zhyncs) [inline docs/en/inference/pipeline.md:35]: It has been merged. So we may update this description. Thanks.

Comment (lvhan028) [reply docs/en/inference/pipeline.md:35]: Sure

File: docs/en/faq.md [hunk 2]
```diff
@@ -41,12 +41,40 @@ export LD_LIBRARY_PATH={Location}/nvidia/nccl/lib:$LD_LIBRARY_PATH
...
+# chat command
+lmdeploy chat turbomind internlm/internlm2-chat-7b --cache-max-entry-count 0.2
+
+# server command
+lmdeploy serve api_server internlm/internlm2-chat-7b --cache-max-entry-count 0.2
+```
 
 ## Serve
 
+### OOM
```
Comment (AllentDan) [inline docs/en/faq.md:76]: Lack of items for OOM?

File: docs/en/inference/pipeline.md (hunk 1 above)
Comment (lvhan028) [reply docs/en/inference/pipeline.md]: Updated. Please review again

**Review by AllentDan — APPROVED** [review]
LGTM

**Review by irexyc — APPROVED** [review]

**Pull request merged**